pkg/controllers/podgroup/pg_controller_handler.go

```diff
@@ -87,15 +88,19 @@ func (pg *pgcontroller) addReplicaSet(obj interface{}) {
 		podList, err := pg.kubeClient.CoreV1().Pods(rs.Namespace).List(context.TODO(),
 			metav1.ListOptions{LabelSelector: metav1.FormatLabelSelector(&selector)})
 		if err != nil {
-			klog.Errorf("Failed to list pods for ReplicaSet <%s/%s>: %v", rs.Namespace, rs.Name, err)
+			klog.Errorf("Failed to list pods for ReplicaSet %s: %v", klog.KObj(rs), err)
 			return
 		}
 		if podList != nil && len(podList.Items) > 0 {
 			pod := podList.Items[0]
-			klog.V(4).Infof("Try to create podgroup for pod %s/%s", pod.Namespace, pod.Name)
+			klog.V(4).Infof("Try to create podgroup for pod %s", klog.KObj(&pod))
```

Comment:
klog.KObj(pod)) is ok: )

Reply:
The value must be a pointer. pod:= podList.Items[0], pod is not a pointer.

Reply:
OK.